src/components/table-hoc/styles/TableWithPredicates.scss

```diff
@@ -0,0 +1,6 @@
+.tablePredicateFilters {}
+
+.tablePredicateFilters + .tablePredicateFilters {
```

Comment:
`mod-no-border-left` and `pl0` can work ?

Reply:
I didn't have the correct behaviour because I only want it when there's two predicate next to each others